defaults/main.yml

```diff
@@ -29,7 +29,6 @@ es_max_open_files: 65536
 es_max_threads: "{{ 2048 if ( es_version is version_compare('6.0.0', '<')) else 8192 }}"
 es_max_map_count: 262144
 es_allow_downgrades: false
-es_xpack_features: ["alerting","monitoring","graph","ml","security"]
```

Comment:
It might be easier to just make this an empty list. It would remove the need to check if it is defined all the time and makes the setting a bit more discoverable.